src/status_im2/contexts/wallet/events.cljs

```diff
@@ -231,18 +232,37 @@
 
 (rf/reg-event-fx :wallet/store-last-collectible-details store-last-collectible-details)
 
+(def collectible-data-types
```

Comment:
It's very helpful during refactors to see private vars with the metadata because then we feel much safer to move things around knowing we're not missing usages. clojure-lsp helps a lot, but I don't fully trust it at all times because it's easy to get in a situation where the cache is stale while changing a lot of code.